<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompositionModeDialogForm</class>
 <widget class="QDialog" name="CompositionModeDialogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Composition Mode</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <widget class="QFrame" name="frame">
       <property name="styleSheet">
        <string notr="true">QWidget { background: white }</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="m_titleLabel">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Defines the composition mode to render this layer.</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="m_imgLabel">
          <property name="minimumSize">
           <size>
            <width>112</width>
            <height>48</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="text">
           <string>Imagem</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Composition Modes</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QRadioButton" name="m_clearRadioButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="m_dstInRadioButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Destination In</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QRadioButton" name="m_plusRadioButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Plus</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QRadioButton" name="m_colorDodgeRadioButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Color Dodge</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="m_srcRadioButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Source</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QRadioButton" name="m_srcOutRadioButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Source Out</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QRadioButton" name="m_multRadioButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Multiply</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QRadioButton" name="m_colorBurnRadioButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Color Burn</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="m_dstRadioButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Destination</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QRadioButton" name="m_dstOutRadioButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Destination Out</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QRadioButton" name="m_screenRadioButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Screen</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QRadioButton" name="m_hardLightRadioButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Hard Light</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="m_srcOverRadioButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Source Over (default)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QRadioButton" name="m_srcAtTopRadioButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Source At Top</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QRadioButton" name="m_overlayRadioButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Overlay</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QRadioButton" name="m_softLightRadioButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Soft Light</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QRadioButton" name="m_dstOverRadioButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Destination Over</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QRadioButton" name="m_dstAtTopRadioButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Destination At Top</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QRadioButton" name="m_darkenRadioButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Darken</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QRadioButton" name="m_diffRadioButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Difference</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QRadioButton" name="m_srcInRadioButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Source In</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QRadioButton" name="m_xorRadioButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>XOR</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QRadioButton" name="m_lightenRadioButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Lighten</string>
            </property>
           </widget>
          </item>
          <item row="5" column="3">
           <widget class="QRadioButton" name="m_exclusionRadioButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Exclusion</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Diagram</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="m_diagramLabel">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>150</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>150</height>
             </size>
            </property>
            <property name="text">
             <string>Imagem</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Description</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QPlainTextEdit" name="m_decriptionPlainTextEdit">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>60</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="4">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="te::qt::widgets::HelpPushButton" name="m_helpPushButton">
         <property name="text">
          <string>Help</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="m_applyPushButton">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="m_closePushButton">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>te::qt::widgets::HelpPushButton</class>
   <extends>QPushButton</extends>
   <header>terralib/qt/widgets/help/HelpPushButton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_clearRadioButton</tabstop>
  <tabstop>m_srcRadioButton</tabstop>
  <tabstop>m_dstRadioButton</tabstop>
  <tabstop>m_srcOverRadioButton</tabstop>
  <tabstop>m_dstOverRadioButton</tabstop>
  <tabstop>m_srcInRadioButton</tabstop>
  <tabstop>m_dstInRadioButton</tabstop>
  <tabstop>m_srcOutRadioButton</tabstop>
  <tabstop>m_dstOutRadioButton</tabstop>
  <tabstop>m_srcAtTopRadioButton</tabstop>
  <tabstop>m_dstAtTopRadioButton</tabstop>
  <tabstop>m_xorRadioButton</tabstop>
  <tabstop>m_plusRadioButton</tabstop>
  <tabstop>m_multRadioButton</tabstop>
  <tabstop>m_screenRadioButton</tabstop>
  <tabstop>m_overlayRadioButton</tabstop>
  <tabstop>m_darkenRadioButton</tabstop>
  <tabstop>m_lightenRadioButton</tabstop>
  <tabstop>m_colorDodgeRadioButton</tabstop>
  <tabstop>m_colorBurnRadioButton</tabstop>
  <tabstop>m_hardLightRadioButton</tabstop>
  <tabstop>m_softLightRadioButton</tabstop>
  <tabstop>m_diffRadioButton</tabstop>
  <tabstop>m_exclusionRadioButton</tabstop>
  <tabstop>m_decriptionPlainTextEdit</tabstop>
  <tabstop>m_applyPushButton</tabstop>
  <tabstop>m_closePushButton</tabstop>
  <tabstop>m_helpPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_closePushButton</sender>
   <signal>released()</signal>
   <receiver>CompositionModeDialogForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>506</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>546</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
